quickfixj-base/src/main/java/quickfix/MessageFactory.java

```diff
@@ -24,7 +24,7 @@
 /**
  * Used by a Session to create a Message.
  *
- * @see quickfix.Session
+ * quickfix.Session
```

Comment:
Same here `@see`

Reply:
That class is in quickfixj-core so the @see tag causes Javadoc warnings or errors, core depends on base not visa versa. I did not want to be aggressive and lose the information that the author intended so I just took away the "@" so that no javadoc errors result.